lib/src/wolt_modal_sheet.dart

```diff
@@ -264,8 +275,12 @@ class WoltModalSheet<T> extends StatefulWidget {
 class WoltModalSheetState extends State<WoltModalSheet> {
   List<SliverWoltModalSheetPage> _pages = [];
 
-  Widget Function(Widget) get _decorator =>
-      widget.decorator ?? (widget) => Builder(builder: (_) => widget);
+  Widget Function(Widget) get _pageContentDecorator =>
+      widget.pageContentDecorator ??
+      (widget) => Builder(builder: (_) => widget);
+
+  Widget Function(Widget) get _modalDecorator =>
+      widget.modalDecorator ?? (widget) => Builder(builder: (_) => widget);
```

Comment:
We could get rid of the default builders and only apply when decorator builders are not null.

To save you time, i will push the commit with the suggestion. Please feel free to revert the commit if not agreed.

Reply:
Here is the applied commit https://github.com/woltapp/wolt_modal_sheet/pull/267/commits/14062818b335b5bb9e76c59a093fef5c2abfa61c

Reply:
@TahaTesser <3 you are amazing!